db.go

```diff
@@ -63,6 +63,11 @@ type DB struct {
 	// https://github.com/boltdb/bolt/issues/284
 	NoGrowSync bool
 
+	// If you want to read the entire database fast, you can set MmapFlag to
+	// syscall.MAP_POPULATE on Linux 2.6.23+ for sequential read-ahead.
+	// https://github.com/coreos/etcd/issues/3786
```

Comment:
Let's remove the issue reference from etcd. It does not make a lot of sense to others.